src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt

```diff
@@ -14,21 +16,24 @@ import uk.gov.justice.digital.hmpps.managerecallsapi.service.PrisonLookupService
 @Component
 class DossierContextFactory(
   @Autowired private val recallRepository: RecallRepository,
-  @Autowired private val prisonLookupService: PrisonLookupService
+  @Autowired private val prisonLookupService: PrisonLookupService,
+  @Autowired private val documentRepository: DocumentRepository,
 ) {
   fun createContext(recallId: RecallId): DossierContext {
     val recall = recallRepository.getByRecallId(recallId)
     val currentPrisonId = recall.currentPrison!!
     val currentPrisonName = prisonLookupService.getPrisonName(currentPrisonId)
     val currentPrisonIsWelsh = prisonLookupService.isWelsh(currentPrisonId)
-    return DossierContext(recall, currentPrisonName, currentPrisonIsWelsh)
+    val newVersion = (documentRepository.findLatestVersionedDocumentByRecallIdAndCategory(recallId, DOSSIER)?.version ?: 0) + 1
```

Comment:
Does this really need to be `newVersion`?  I don't see why simply `version` isn't OK?